IBAnimatableApp.xcodeproj/project.pbxproj

```diff
@@ -920,7 +926,7 @@
 		AE6B015F1C2A082300950BB2 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				APPLICATION_EXTENSION_API_ONLY = YES;
+				APPLICATION_EXTENSION_API_ONLY = NO;
```

Comment:
Why do we need to change that? It will break the user use IBAnimatable in WatchKit and Today view